packages/flame/lib/src/components/component_set.dart

```diff
@@ -90,10 +98,17 @@ class ComponentSet extends QueryableOrderedSet<Component> {
     Iterable<Component> components, {
     BaseGame? gameRef,
```

Comment:
`gameRef` no longer needed here

Reply:
Ah, haven't pushed the latest commits.
Its still quite some work left, especially with the effects.

Reply:
This one I had apparently forgotten about, good catch!